File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        // a ⊕ b를 계산
        int concatValue = Integer.parseInt(String.valueOf(a) + String.valueOf(b));

        // 2 * a * b를 계산
        int productValue = 2 * a * b;

        // 두 값 중 더 큰 값을 반환, 같으면 concatValue를 반환
        return Math.max(concatValue, productValue);
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.py
def solution(a, b):
    answer = 0

    ab = str(a) + str(b)
    ba = 2 * a * b

    if int(ab) > int(ba):
        answer = int(ab)
    elif int(ba) > int(ab):
        answer = int(ba)
    elif int(ab) == int(ba):
        answer = int(ab)

    return answer